README.md

```diff
@@ -20,3 +20,4 @@ the `#team-software` or `#team-analysis` channels).
 - [Setting up your terminal](terminal.md)
 - [Data sharing](data_sharing.md)
 - [R style guide](r_style_guide.md)
+- [cpg-utils config guide](cgp-utils_config.md)
```

Comment:
I understand that the library is called `cpg-utils`, but might be easier for consistency to keep underscores in file names instead of mixing hyphens + underscores.

```suggestion
- [`cpg-utils` config guide](cgp_utils_config.md)
```